packages/flutter/lib/rendering.dart

```diff
@@ -27,6 +27,7 @@ export 'package:flutter/foundation.dart' show
   ValueChanged,
   ValueGetter,
   ValueSetter;
+export 'package:flutter/foundation.dart' show DiagnosticLevel;
```

Comment:
why not add this to the list above?

Reply:
done. In other files I was seeing repeated export statements for blocks of names that weren't conceptually related. I don't have a preference either way.